controllers/podlabels/pod.go

```diff
@@ -0,0 +1,26 @@
+package podlabels
+
+import (
+	"context"
+	"github.com/go-logr/logr"
+	corev1 "k8s.io/api/core/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+type PodLabelsReconciler struct {
+	abstractPodLabelsReconciler
+
+	Log logr.Logger
+}
```

Comment:
I think we don't need the abstract here, since the Pod API is just a single one.